client/components/navbar.js

```diff
@@ -2,22 +2,50 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { logout } from '../store';
+import history from '../history';
+import { fetchUserFunction, addUserFunction } from '../crud/user';
+const { db } = require('../../firebase/initFirebase');
 
 const { auth } = require('../../firebase/initFirebase');
 
+const firebase = require('firebase/app');
+require('firebase/firestore');
+require('firebase/auth');
+
 export default class Navbar extends Component {
   constructor(props) {
     super(props);
     this.state = {
       isLoggedIn: false, // boolean, able to access user off auth.currentUser
+      displayName: 'guest',
     };
   }
 
   componentDidMount() {
     auth.onAuthStateChanged((user) => {
       if (user) {
         // if there is a user logged in, change state of isLoggedIn to true
-        this.setState({ isLoggedIn: true });
+        this.setState({
+          isLoggedIn: true,
+          displayName: firebase.auth().currentUser.displayName,
+        });
+
+        const { uid } = user;
+        db.collection('users').doc(uid).get()
+        .then(userExists => {
```

Comment:
So the rule for AirBnB is when you have curly braces with an arrow function, you should put parentheses around the parameter, but if don't have curly braces you can omit the parentheses around the parameter for the arrow function.
https://github.com/airbnb/javascript#arrows--one-arg-parens